runtime/lua/vim/lsp/buf.lua

```diff
@@ -683,11 +683,7 @@ local function on_code_action_results(results, ctx, options)
     --
     local client = vim.lsp.get_client_by_id(action_tuple[1])
     local action = action_tuple[2]
-    if
-      not action.edit
-      and client
-      and vim.tbl_get(client.server_capabilities, 'codeActionProvider', 'resolveProvider')
-    then
+    if not action.edit and client and client.supports_method('codeAction/resolve') then
```

Comment:
Turns out this change doesn't work.
`client.supports_method('codeAction/resolve')` returns `false` if it's supported via dynamic capabilities because there won't be an entry for `codeAction/resolve`, but instead it looks like this:

```
        ["textDocument/codeAction"] = { {
            id = "3b2131ee-bf17-4a3d-a528-ad698f162956",
            method = "textDocument/codeAction",
            registerOptions = {
              codeActionKinds = { "quickfix", "refactor", "refactor.extract", "refactor.inline", "refactor
.rewrite", "source", "source.organizeImports" },
              resolveProvider = true
            }
          } },
```

Using code actions which require resolving the command is currently broken on master because of this.

Reply:
See https://github.com/neovim/neovim/pull/23826